packages/expo-cli/src/commands/start.js

```diff
@@ -219,6 +258,12 @@ export default (program: any) => {
     )
     // TODO(anp) set a default for this dynamically based on whether we're inside a container?
     .option('--max-workers [num]', 'Maximum number of tasks to allow Metro to spawn.')
+    .option('--dev', 'Turns dev flag on')
```

Comment:
```suggestion
    .option('--dev', 'Turns the development mode flag on')
```

Reply:
Maybe clearer to say `turns development mode on/off`?

Reply:
sounds good to me